data/boarding missions.txt

```diff
@@ -60,6 +60,77 @@ mission "Assisting Merchant (Large)"
 	destination "Earth"
 
 
+mission "Asteroid Course"
+	assisting
+	minor
+	name "Passengers to <planet>"
+	description "Transport the remaining crew of a destroyed ship, <bunks> people that were working on the <origin>, to <destination>."
+	to offer
+		random < 6
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	passengers 3 7
+	destination 
+		distance 1
+		not government "Pirate"
```

Comment:
Should the destination include `attributes shipyard`?

Reply:
added